src/audio/google/google_rtc_audio_processing.c

```diff
@@ -338,6 +389,10 @@ static struct comp_dev *google_rtc_audio_processing_create(
 	if (!cd)
 		goto fail;
 
+#if CONFIG_IPC_MAJOR_4
+	cd->config = *(struct sof_ipc4_aec_config *)spec;
```

Comment:
I understand that we're just reading the data, but for the sake of documentation for developers, reading and potentially modifying this code I'd make the cast preserve the `const` qualifier, i.e.
```
cd->config = *(const struct sof_ipc4_aec_config *)spec;
```

Reply:
thanks, done!